build/circle-test.sh

```diff
@@ -77,8 +77,18 @@ prepare_artifacts() {
         -d "${2}"
       }
 
+      echo "Posting an issue"
       # Generate string of failed tests: 'TestRaftRemoveRace TestChaos TestHoneyBooBoo'
-      FAILEDTESTS=$(grep -oh '^--- FAIL: \w*' "${outdir}/excerpt.txt" | sed -e 's/--- FAIL: //' | tr '\n' ' ')
+      FAILEDTESTS=$(grep -oh '^--- FAIL: \w*' "${outdir}/excerpt.txt" | sed -e 's/--- FAIL: //' | tr '\n' ' ' || true)
```

Comment:
Do we even need this line any more? Is there any case where it would work but we wouldn't get the answer from CIRCLE_TEST_REPORTS? (aside from the fact that grep is not the best way to parse xml?)